contracts/0.8.9/interfaces/IStakingModule.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: 2022 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.9;
+
+/// @title Lido's Staking Module interface
+interface IStakingModule {
+    /// @notice Returns the type of the staking module
+    function getType() external view returns (bytes32);
+
+    /// @notice Returns the validators stats of all node operators in the staking module
+    /// @return exitedValidatorsCount Total number of validators in the EXITED state
+    /// @return activeValidatorsKeysCount Total number of validators in active state
+    /// @return readyToDepositValidatorsKeysCount Total number of validators ready to be deposited
+    function getValidatorsKeysStats()
```

Comment:
Validator~~s~~Keys => ValidatorKeys

(here and throughout the module)

Reply:
fixed